### PR TITLE
feat(rest): Add upload description to trigger fossology process

### LINKED ISSUE #1533: Need API support for Updation of component name,version and sw360 link in Fossology
(from sw360 user)

Hi,
Recently we tried out the trigger fossology process api provided in the SW360 Rest Api documentation.
'https://sw360.org/api/releases/987456/triggerFossologyProcess?markFossologyProcessOutdated=false'
this feature is very helpful for the component upload to fossology directly from sw360.
But as an addition to this it would be helpful if we are able to update the description field of fossology upload with the respective component Name, Version and the link to release page in sw360.
Kindly support this feature as well in the same api.
Thanks in advance.

### PR DESCRIPTION
**Pull request by JaideepPalit**
Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

> Add upload description to trigger fossology process.
> * Which issue is this pull request belonging to and how is it solving it? (*#1533*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: closes #1533 

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

### CODE REVIEW

File: rest/resource-server/src/docs/asciidoc/releases.adoc [hunk 1]
```diff
@@ -239,7 +239,9 @@ have been successful.
 ==== Trigger FOSSology process
 
 A `GET` request will allow you to Trigger FOSSology process for a release. +
-Please set the request parameter `&markFossologyProcessOutdated=<true|false>` to set previous FOSSology process outdated and generate new.
+Please set the request parameter  +
+   `&markFossologyProcessOutdated=<true|false>` to set previous FOSSology process outdated and generate new. +
+   `&uploadDescription=uploadDescription` with the upload description to FOSSology +
```
Comment (smrutis1) [inline rest/resource-server/src/docs/asciidoc/releases.adoc:244]: Please remove the `+` at the end. I think it's not necessary.

**Review by smrutis1 — COMMENTED** [review]
One minor review comment

File: rest/resource-server/src/docs/asciidoc/releases.adoc (hunk 1 above)
Comment (JaideepPalit) [reply rest/resource-server/src/docs/asciidoc/releases.adoc:244]: `+` is added to add newline/line break. Without it comes in a single line
Shall I remove it?

**Review by ag4ums — APPROVED** [review]
working as expected

**Pull request merged**